pkg/local-disk-manager/handler/localdisk/localdisk.go

```diff
@@ -191,6 +191,7 @@ func (ldHandler *Handler) FilterDisk(ldc *v1alpha1.LocalDiskClaim) bool {
 		Init().
 		Available().
 		HasNotReserved().
```

Comment:
what will happen if localdiskname is matched but other attrributes are not? is this disk can be assigned directly to a disk claim?

Reply:
No, the logic of all conditions is AND.